Source/WTF/wtf/Vector.h

```diff
@@ -1121,15 +1121,19 @@ template<typename T, size_t inlineCapacity, typename OverflowHandler, size_t min
 template<typename U>
 bool Vector<T, inlineCapacity, OverflowHandler, minCapacity, Malloc>::contains(const U& value) const
 {
-    return find(value) != notFound;
+    for (const auto& elem : *this) {
+        if (value == elem)
+            return true;
+    }
+    return false;
 }
 
 template<typename T, size_t inlineCapacity, typename OverflowHandler, size_t minCapacity, typename Malloc>
 template<typename MatchFunction>
 size_t Vector<T, inlineCapacity, OverflowHandler, minCapacity, Malloc>::findIf(const MatchFunction& matches) const
 {
     for (size_t i = 0; i < size(); ++i) {
-        if (matches(at(i)))
+        if (matches(data()[i]))
```

Comment:
This is less safe since it bypasses the bounds check. We should never bypass bounds checking unless there is a proven perf benefit.